app/v2/notifications/post_notifications.py

```diff
@@ -249,6 +253,27 @@ def post_notification(notification_type):
     return jsonify(resp), 201
 
 
+# Priority lanes feature (FF_PRIORITY_LANES)
```

Comment:
instead of this comment, can you put a docstring below (if you want to leave a comment)
https://stackoverflow.com/questions/3898572/what-are-the-most-common-python-docstring-formats (either use the google format/ or the numpy one I thing)

Reply:
👍